lib/models/transform.js

```diff
@@ -211,9 +212,13 @@ class Transform extends new Record(DEFAULT_PROPERTIES) {
 
     if (includes(DOCUMENT_TRANSFORMS, type)) {
       let { document, selection } = state
-      let [ range, ...rest ] = args
-      range = range.normalize(document)
-      document = document[type](range, ...rest)
+      try {
```

Comment:
Let's create another array constant here so that we end up with `DOCUMENT_RANGE_TRANSFORMS` and `DOCUMENT_NODE_TRANSFORMS` so that we don't have to `try/catch`.